compiler/luci-interpreter/src/core/KernelParams.h

```diff
@@ -19,6 +19,7 @@
 
 #include <luci/IR/AttrPadding.h>
 #include <luci/IR/AttrFusedActFunc.h>
+#include <loco/IR/DataType.h>
```

Comment:
Please use `luci_interpreter/DataType.h`, there is already `DataType`.

Reply:
Okay! will fix this!